src/main/java/kitchenpos/eatinorder/domain/model/todo/OrderTableOccupiedState.java

```diff
@@ -0,0 +1,44 @@
+package kitchenpos.eatinorder.domain.model.todo;
+
+import java.util.Objects;
+
+public class OrderTableOccupiedState {
+    private static final OrderTableOccupiedState OCCUPIED = new OrderTableOccupiedState(true);
+    private static final OrderTableOccupiedState VACANT = new OrderTableOccupiedState(false);
+    private final boolean occupied;
```

Comment:
용어사전에 정의된 영문명을 사용해보면 어떨까요?
- 빈테이블: Empty Table
- 사용중: Used

```suggestion
private static final OrderTableOccupiedState USED = new OrderTableOccupiedState(true);
    private static final OrderTableOccupiedState EMPTY = new OrderTableOccupiedState(false);
```

Reply:
넵 기존 변수명 수정할 생각을 못하고 있었습니다!